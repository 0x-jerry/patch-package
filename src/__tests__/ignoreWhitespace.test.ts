import { runShellTest } from "./shellTests"

runShellTest("ignore-whitespace")
