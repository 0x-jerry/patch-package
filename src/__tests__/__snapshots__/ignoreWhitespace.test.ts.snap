// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ignore-whitespace: empty changeset when adding whitespace 1`] = `
"SNAPSHOT: empty changeset when adding whitespace
⁉️  Not creating patch file for package 'alphabet'
⁉️  There don't appear to be any changes.
END SNAPSHOT"
`;

exports[`Test ignore-whitespace: line a changed 1`] = `
"SNAPSHOT: line a changed
patch-package
--- a/node_modules/alphabet/index.js
+++ b/node_modules/alphabet/index.js
@@ -1,4 +1,4 @@
-// a
+// patch-package
 // b   
 // c 
 // d
END SNAPSHOT"
`;
